Sources/SPTDataLoaderRequestTaskHandler.h

```diff
@@ -73,6 +73,10 @@ NS_ASSUME_NONNULL_BEGIN
  @return The response object unless the request was cancelled or will be re-tried in which case `nil` is returned.
  */
 - (nullable SPTDataLoaderResponse *)completeWithError:(nullable NSError *)error;
+/**
+ Called when a request with waitsForConnectivity enters the waiting state
+*/
+- (void)noteWaitingForConnectivity;
```

Comment:
This isn’t a very good name, but because `SPTDataLoaderRequestTaskHandler` doesn’t follow the delegate naming convention it’s hard to come up with a good verb form.